platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/exceptions/TelemetryServiceNotConfiguredException.java

```diff
@@ -12,7 +12,7 @@
 public class TelemetryServiceNotConfiguredException extends RuntimeException {
 
     public TelemetryServiceNotConfiguredException() {
-        super("Using Mapbox Android SDK requires configuring TelemetryService. See the INSTALL.md");
+        super("\nUsing Mapbox Android SDK requires configuring TelemetryService. This only requires two steps:\n    1. Include the WAKE_LOCK permission within your applications AndroidManifest.xml\n    2. Add the \"com.mapbox.mapboxsdk.telemetry.TelemetryService\" service in your applications AndroidManifest.xml\n    For more help, visit https://www.mapbox.com/android-sdk/");
     }
```

Comment:
@bleege we should get rid of the service altogether.  Instead just convert it into a regular class hooked with life cycle of the MapView.